docker/tap-schema/1015_dp01_dc2_tap_schema.sql

```diff
@@ -0,0 +1,1185 @@
+INSERT INTO tap_schema.schemas11 (schema_name, utype, description, schema_index) VALUES ('dp01_dc2_catalogs.dp01_dc2_catalogs', null, 'Data Preview 0 includes five tables based on the DESC’S Data Challenge 2 simulation of 300 square degrees of the wide-fast-deep LSST survey region after 5 years. All tables contain objects detected in coadded images.', 0);
+INSERT INTO tap_schema.tables11 (schema_name, table_name, table_type, utype, description, table_index) VALUES ('dp01_dc2_catalogs.dp01_dc2_catalogs', 'dp01_dc2_catalogs.dp01_dc2_catalogs.position', 'table', null, 'Select astrometry-related parameters for objects detected in the coadded images, such as coordinates, footprints, patch/tract information, and deblending parameters. (29 columns)', 0);
```

Comment:
See above schema-name comment.  I won't comment on all the other `tables11` `INSERT` commands below, though.